fms2_io/netcdf_io.F90

```diff
@@ -149,6 +149,11 @@ module netcdf_io_mod
   character (len=20) :: time_name
   type(dimension_information) :: bc_dimensions !<information about the current dimensions for regional
                                                !! restart variables
+  logical :: use_collective = .false. !< Flag indicating if we should open the file for collective input
+                                      !! this should be set to .true. in the user application if they want
+                                      !! collective reads (put before open_file())
+  integer :: tile_comm=MPP_COMM_NULL   !< MPI communicator used for collective reads.
+                                      !! To be replaced with a real communicator at user request
```

Comment:
How are these set?

Reply:
Previous testing with and without the io_layout=1,1 'fix' did not reveal anything.  I should probably redo that testing.

As for how those are set, the user needs to set those in their application.

I have attached a 'patch' file for atmos_cubed_sphere/tools/external_ic.F90 as an example.
[external.patch](https://github.com/NOAA-GFDL/FMS/files/14727702/external.patch)

I added similar changes to 
FV3/atmos_cubed_sphere/tools/fv_io.F90 (fv_core.res,fv_tracer.res)
FV3/io/fv3atm_restart_io.F90 (Oro_restart, Sfc_restart, Phy_restart)